pilot/pkg/networking/util/util.go

```diff
@@ -33,6 +33,11 @@ import (
 	"istio.io/istio/pkg/log"
 )
 
+const (
+	// Blackhole cluster to catch traffic from routes with unresolved clusters. Traffic arriving here goes nowhere.
+	BlackholeCluster = "blackholeCluster"
```

Comment:
Can we actually put this somewhere meaningful inside of the v1alpha3 package? Having this util/util.go package is actually an anti-pattern.